packages/kit/test/apps/hash-based-routing/src/routes/+layout.js

```diff
@@ -0,0 +1,9 @@
+export const ssr = false;
+
```

Comment:
I think this should be mandatory in hash mode — the server can't know which page to render, so it shouldn't render anything (in production we can use a prerendered fallback page). Once that's done, we shouldn't need to specify it here

```suggestion
```